src/uk/ufdub/src/eu/kanade/tachiyomi/animeextension/uk/ufdub/UFDub.kt

```diff
@@ -0,0 +1,10 @@
+package eu.kanade.tachiyomi.animeextension.uk.ufdub
+
+import eu.kanade.tachiyomi.animesource.AnimeSource
+import eu.kanade.tachiyomi.animesource.AnimeSourceFactory
+
+class UFDub : AnimeSourceFactory {
```

Comment:
Why did you make a factory if there is only one source?

Reply:
Based on AnimeVost(RU)
I can remove